client/Player/components/PlayerView.tsx

```diff
@@ -25,6 +27,7 @@ interface LocalState {
 interface OwnProps {
   onSuggestDamage: ApplyDamageCallback;
   onSuggestTag: (combatantId: string, tagState: TagState) => void;
+  combatStats: CombatStats;
```

Comment:
You could make this property optional here:
`combatStats?: CombatStats;`
This allows it to be `undefined`, so you could omit most of the lines in tests, etc. where you're setting it to `null`.